app/lib/service/download_counts/sync_download_counts.dart

```diff
@@ -143,7 +148,7 @@ Future<void> syncDownloadCounts() async {
     final syncDate = today.addCalendarDays(-i);
     final fileName = [
       'daily_download_counts',
-      '${syncDate.year}-${syncDate.month}-${syncDate.day}T00:00:00Z',
+      formatDateForFileName(syncDate),
```

Comment:
Another option would be to do
```suggestion
      DateTime(syncDate.year, syncDate.month, syncDate.day).toIso8601String(),
```

Reply:
Are you sure this works? I think it ends with `000` and not `00Z`